diag_manager/fms_diag_object.F90

```diff
@@ -206,8 +206,8 @@ integer function fms_register_diag_field_obj &
 #else
  diag_field_indices = find_diag_field(varname, modname)
  if (diag_field_indices(1) .eq. diag_null) then
-    !< The field was not found in the table, so return diag_null
-    fms_register_diag_field_obj = diag_null
+    !< The field was not found in the table, so return DIAG_FIELD_NOT_FOUND
+    fms_register_diag_field_obj = DIAG_FIELD_NOT_FOUND
```

Comment:
The doxygen \\return needs to be updated to reflect this

Reply:
562bb3e9e2239db8ed5087363b966117f6ca28bb